File: Java/UberVan.java
import java.util.ArrayList;
import java.util.Map;

//Clase UberVan (Hereda de la clase Carro)
class UberVan extends Car {
    // Atributos o propiedades de la clase
    private Integer id;
    private Map<String, ArrayList<String, Integer>> typeCarAccepted;
    private ArrayList<String> seatsMaterial;

    // Método constructor
    public UberVan(String license, Driver driver, Map<String, ArrayList<String, Integer>> typeCarAccepted,
            ArrayList<String> seatsMaterial) {
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }
}

File: PHP/UberVan.php
<?php
require_once('Car.php');
//Clase UberVan (Hereda de la clase Carro)
class UberVan extends Car
{
    // Atributos o propiedades de la clase
    public $typeCarAccepted;
    public $seatsMaterial;

    // Método constructor
    public function __construct($license, $driver, $typeCarAccepted, $seatsMaterial)
    {
        parent::__construct($license, $driver);
        $this->typeCarAccepted = $typeCarAccepted;
        $this->seatsMaterial = $seatsMaterial;
    }
}